libraries/chaindb/.github/workflows/build.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Build & Test"
```

Comment:
Do we need this anymore? Seems like these should be just part of Spring now.

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/3f7cfc65b654e4866522fd20ba4b86ded66b5387